src/main/java/cart/domain/CartItem.java

```diff
@@ -5,24 +5,62 @@
 import java.util.Objects;
 
 public class CartItem {
+
+    public static final int MINIMUM_QUANTITY = 1;
+
     private Long id;
     private int quantity;
     private final Product product;
     private final Member member;
 
     public CartItem(Member member, Product product) {
         this.quantity = 1;
```

Comment:
C : Defualt 값을 상수로 주입해도 좋을거 같아
```suggestion
        this.quantity = MINIMUM_QUANTITY;
```

Reply:
반영해뒀어!